src/main/java/com/marginallyclever/makelangelo/select/SelectReadOnlyText.java

```diff
@@ -2,56 +2,83 @@
 
 import java.awt.BorderLayout;
 import java.awt.Desktop;
-import java.awt.event.MouseAdapter;
-import java.awt.event.MouseEvent;
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
+import javax.swing.JEditorPane;
 
-import javax.swing.JLabel;
+import javax.swing.ToolTipManager;
+import javax.swing.event.HyperlinkEvent;
+import javax.swing.event.HyperlinkListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * Read only HTML-rich text that can display AT MOST one clickable link.
- * @author Dan Royer
+ * Read only JEditorPane with an HyperLinckListener for text/html contening or not multiple html link.
+ * With clickable link (Desktop.BROWSE) and ToolTips when hovered over .
+ * 
+ * @author Dan Royer (and modified by PPAC37)
  * @since 7.24.0
  */
 public class SelectReadOnlyText extends Select {
 	/**
 	 * 
 	 */
-	private static final long serialVersionUID = -8918068053490064344L;
-	private static final String A_HREF = "<a href=";
```

Comment:
will break if someone does something dumb like "<a     href    ="

Reply:
I hadn't paid attention, I guess with the JEditorPane and HyperlinkListener this should no longer be a problem.